platform/server/pom.xml

```diff
@@ -197,6 +197,12 @@
             <groupId>org.postgresql</groupId>
             <artifactId>org.motechproject.org.postgresql</artifactId>
         </dependency>
+        <!-- Active MQ Version -->
+        <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>activemq-all</artifactId>
+            <version>5.11.0</version>
```

Comment:
1) Versions go to the dependency management in the main pom
2) Do we need this? Don't we already have a different amq dependency?